### DISCUSSION TITLE
404 Page Not Found Browser Error

### DISCUSSION THREAD

**Discussion by wci-aw** [YOURLS/YOURLS#3761] [General]
Not sure why, but getting  server 404 error since yesterday on my short links in use.

Short linke should https://2share.me/test-url > forward to > corresponding page which is online
But instead I'm getting the browser error 404 

![image](https://github.com/YOURLS/YOURLS/assets/8851682/e5213215-545f-4ef0-a175-35b8d85cbbe0)

Any help would be apreshiated. 

Best regards, Alex

**Comment by LeoColomb** [comment]
@wci-aw This screenshot shows a server error (as written on first line), not a browser error.
Have you contacted your host supplier support? What is their answer?
What are the logs?

**Reply by dgw** [reply]
Looks like the number one issue on our troubleshooting page. https://yourls.org/docs/guide/troubleshooting/frequent-issues#404-instead-of-redirection

**Comment by wci-aw** [comment]
The server support cannot find anything so far.
Here is the log
![image](https://github.com/YOURLS/YOURLS/assets/8851682/0bf3e64a-ec67-4554-ae8c-c183b0fa78d7)


I have not changed any server settings exept 
changed the outdated PHP version but even when changing back, its not working
![image](https://github.com/YOURLS/YOURLS/assets/8851682/f97750e5-378f-4eef-80b4-b07f93b7b1d4)

Does it has to do with permission rights?
![image](https://github.com/YOURLS/YOURLS/assets/8851682/018ce08d-54f7-4df5-9924-42abadea69e6)

Looking forward to hear from you.

Best, Alex

**Comment by wci-aw** [comment]
AH one more thing I remember I've done.
I've installed this plugin - https://github.com/joshp23/YOURLS-URL-Health-Check
does this maybe has to do with the damage?

I've deinstalled the plugin and reinstalled yours before I added this plugin, but still facing the same issue.

Best, Alex

**Comment by wci-aw** [comment]
P.S. I have another yourls installation which I did'nt touched on this server and having the same issue. Not sure how to solve, as the server admins are saying its not their fault or cannot find anything. The only thing I need to mention is, that they have changed their hosting 1 week ago but it was working after this and now for any reason since yesterday not anymore. The name of my host is "hostlegends"

**Comment by wci-aw** [comment]
Hi,
I could have solve it now by adding a .htaccess file into the root folder.
The strange thing is, it has worked formerly without having such a file existing.
BR, Alex

**Reply by dgw** [reply]
It's good you were able to figure it out. The `.htaccess` file might not have been needed before if domain configuration was managed elsewhere, or it might have been deleted by accident in the mentioned "changed their hosting 1 week ago" event (perhaps files were migrated with a process that skips dotfiles by default, like rsync without [setting `dotglob`](https://www.gnu.org/software/bash/manual/html_node/The-Shopt-Builtin.html)).
